sympy/ntheory/ec.py

```diff
@@ -0,0 +1,360 @@
+from sympy.abc import x, y
```

Comment:
These should not be global variables. They should be defined locally by something like `x, y = symbols('x y')` if they are needed.